src/containers/AccountConnection.jsx

```diff
@@ -220,7 +216,10 @@ class AccountConnection extends Component {
 
   render () {
     const { t, existingAccount, connector, fields } = this.props
-    const { submitting, deleting, error, credentialsError, isSuccessTimedOut } = this.state
+    const { submitting, deleting, error, isSuccessTimedOut } = this.state
+    const hasGlobalError = error && error.message && ![
+      ACCOUNT_ERRORS.LOGIN_FAILED
+    ].includes(error.message)
```

Comment:
@lespacedunmatin You're right about `error.message !== ACCOUNT_ERRORS.LOGIN_FAILED`. However, `indexOf` returns `-1` if the given object is not contained in the given array, which is a truthy value.
```js
console.log(-1 ? true : false)
// true
```